### ISSUE TITLE
Static IP setting in 40G VM auto installation script not working

### ISSUE THREAD

**Issue by mz78** [nextcloud/vm#1567]
## Steps to reproduce
Download pre-installed 40G vm template from hanssonit.com.
1. Import 40G vm to my ESXi.
2. No DHCP server in the VLAN.
3. Start up server and wait for script running, till set up IP address manually.
4. Input my IP (192.168.30.50/24), gateway (192.168.30.1), and my DNS.
5. Confirm the setting correct or not, IP and gateway message disappeared, just [/24] .

## Expected behaviour
The static network setting should be saved after manual configuration.

## Actual behaviour
Static IP could not be saved after setting.

## Server configuration

**Nextcloud server version:** (see your admin page)

**Server OS** (Ubuntu server is default)

**How did you install the VM?** (Scripted install from master **OR** Released version)

## Network
**Do you use DHCP?**
No.
**Is port 80 and/or 443 open?**
Yes
#### Log file (/var/log/nextcloud/nextcloud.log)
```
Insert your nextcloud.log file here
```

#### Installation log (command output)
```
Insert the CLI output here
```

**Comment by szaimen** [comment]
@enoch85 any idea?

**Comment by enoch85** [comment]
I tried this yesterday on the released 20.0.0 version, and all was good. I have no clue.

Could you please try again @mz78?
